File: android/src/withAutostart/java/BootUpReceiver.java
import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import org.ligi.gobandroid_hd.ui.GobanDroidTVActivity;


public class BootUpReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Intent i = new Intent(context, GobanDroidTVActivity.class);
        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        context.startActivity(i);
    }
}

File: android/src/withAutostart/java/BootUpReceiver.kt
package org.ligi.gobandroid_hd

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import org.ligi.gobandroid_hd.ui.GobanDroidTVActivity

class BootUpReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent?) {
        val i = Intent(context, GobanDroidTVActivity::class.java)
        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
        context.startActivity(i)
    }
}